iotedgedev/modules.py

```diff
@@ -128,9 +128,18 @@ def build_push(self, no_build=False, no_push=False):
                         if not no_push:
                             # PUSH TO CONTAINER REGISTRY
                             self.output.info("PUSHING DOCKER IMAGE TO: " + image_destination_name)
+                            # FIND ENV VAR KEY BASED ON REGISTRY VALUE
+                            registry_key = None
+                            for registry in self.envvars.CONTAINER_REGISTRY:
```

Comment:
Should this look up be in the ContainerRegistry class?

Reply:
Then you would just to a cr = ContainerRegistry.get(key)

and then later.

cr.Server
cr.Password
cr.Username